lib/GridItem.jsx

```diff
@@ -218,6 +223,7 @@ export default class GridItem extends React.Component<Props, State> {
     // use this optimization.
     if (this.props.children !== nextProps.children) return true;
     if (this.props.droppingPosition !== nextProps.droppingPosition) return true;
+    if (this.state.allowedToDrag !== nextState.nextState) return true;
```

Comment:
Seems a typo here?

Reply:
Oh, sorry😅!
I meant `this.state.allowedToDrag !== nextState.allowedToDrag`
I'll fix it.